### PR TITLE
Improve help output

### PR DESCRIPTION
**Pull request by sergioc**
- Place commands in code block to improve readability
- Clarify behavior of `Pbot move issue`

### CODE REVIEW

File: src/PBot/Issues/CheckIssuesInMyRepos.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
     public class CheckIssuesInMyRepos : BotCommand
     {
         public CheckIssuesInMyRepos()
-            : base("check my repos$", "pbot check my repos - List the issue that needs attention for repos that you are the caretaker for")
+            : base("check my repos$", "`pbot check my repos` - List the issue that needs attention for repos that you are the caretaker for")
```
Comment (andreasohlund) [inline src/PBot/Issues/CheckIssuesInMyRepos.cs]: @distantcam wonder if we can make the BotCommand smarter here and auto format the help test if we provide the arguments in a more structured way?

Comment (adamralph) [inline src/PBot/Issues/CheckIssuesInMyRepos.cs]: I agree. It's painful to repeat this formatting in every command. We should split the argument in two, e.g. `syntax` and `description`.

File: src/PBot/Issues/MoveIssue.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ public class MoveIssue : BotCommand
     {
         public MoveIssue()
             : base("move issue (.*) to (.*)$",
-            "pbot move issue <repository>#<issue number>(or https://github.com/Particular/repository/issues/issuenumber) to <target repository>(or https://github.com/Particular/repository) - Moves an issue from one repository to the other.")
+            "`pbot move issue <repository>#<issue number>(or https://github.com/Particular/repository/issues/issuenumber) to <target repository>(or https://github.com/Particular/repository)` - Moves an issue from one repository to the other. Comments aren't moved, so revising description of new issue is advised.")
```
Comment (adamralph) [inline src/PBot/Issues/MoveIssue.cs]: @sergioc I've found that comments are usually moved.

However, it did seem to be failing intermittently on some comments when you were moving issues last week.

**Comment by sergioc** [comment]
@adamralph according to @andreasohlund, [it's not meant to move comments anymore](https://github.com/Particular/PBot/issues/50#issuecomment-139511113).

**Comment by adamralph** [comment]
@sergioc ok, thanks. News to me!

**Comment by adamralph** [comment]
Merging because the work is done, but I do think we should refactor as per https://github.com/Particular/PBot/pull/51#discussion_r39264400

**Comment by adamralph** [comment]
@sergioc I released the new version of pbot (0.1.144) but only half of the command in pbot help have the new formatting so I guess you didn't cover them all. It's a step forward, but I guess we really should do the refactoring to split the syntax into a separate parameter and then that will force us to fix the output for all the remaining commands.

**Comment by andreasohlund** [comment]
Also remove the `pbot` prefix, since all commands should be run on a
private conv with the bot?

**Comment by sergioc** [comment]
@andreasohlund :+1:

@adamralph wouldn't it be better for me to run through the commands again and submit a new PR to have it proper while we wait for the refactoring to be done?

**Comment by adamralph** [comment]
@sergioc it would probably be better to just go ahead and do the refactoring.

**Comment by sergioc** [comment]
@adamralph assuming it happens reasonably quickly, I agree :)

**Pull request merged**